### DISCUSSION TITLE
Ender 3D with 4.2.7 board.. Jyers manual mesh 3x3 not homing.

### DISCUSSION THREAD

**Discussion by browsetm** [Jyers/Marlin#1138] [Q&A] [answered]
I have Ender 3 with 4.2.7 board.
E3v2 manual mesh 5x5 works well, however I want to use 3DV2 manual mesh 3x3 unfortunately this version does not home my printer.  help advice would be helpful.

**Comment by Cal-LF** [comment, marked as answer]
You may be downloading the wrong version. Not homing is what happens when the wrong motherboard is selected.

**Reply by browsetm** [reply]
Thanks,  You are probably correct, however just discovered X connector had become unplugged.  Silly error.   Might get glue gun out.

**Reply by Jyers** [reply]
Glue is always the solution 😉

**Reply by browsetm** [reply]
Oh Yes.  My lidl glue gun gets put to use today..
